src/index.jsx

```diff
@@ -368,6 +368,7 @@ export default class ImagesUploader extends Component {
 					const err = {
 						message: 'server error',
 						status: response ? response.status : false,
+                        response,
```

Comment:
as far as I can see, this line uses another indentation guidelines. please, replace it with tabs. otherwise, seems like a good idea to me

Reply:
@kirillDanshin changed it to tabs. Thx for considering it

Reply:
@kirillDanshin anything else?